### PR TITLE
Add credit card documentation, under Billing.

### PR DESCRIPTION
**Pull request by duduzis**
Credit card documentation. Need someone to review it and send me some feedback.

### CODE REVIEW

File: source/billing/credit-card.rst [hunk 1]
```diff
@@ -0,0 +1,119 @@
+###################
+Credit Card payment
+###################
+
+.. note::
+   Only users with the Project Admin role will be able to view and modify the
+   credit card panel
```
Comment (Adrian-Turjak) [inline source/billing/credit-card.rst:7]: Only users with the Project Admin role will be able to view and modify the billing details

File: source/billing/credit-card.rst [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+You and/or your billing contact will still receive a usage summary from us each month with your usage details and a confirmation that your invoice has been paid.
+
+Adding and switching to credit card payment
+===========================================
+
+To setup a credit card, please login_ to the dashboard and follow these steps:
+
+1) Go to Account Details, under Management
+2) Select the Project Billing Details panel
```
Comment (Adrian-Turjak) [inline source/billing/credit-card.rst:20]: tab, not panel

File: source/billing/credit-card.rst [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+.. image:: resources/billing-details-updated.png
+   :align: center
+
+
+Adding a credit card
+====================
+
+.. note::
+   If you have credit card payment as your selected method. You are not allowed to delete
+   the selected credit card.
```
Comment (Adrian-Turjak) [inline source/billing/credit-card.rst:61]: You cannot delete a credit card that is being used by one of your projects.

File: source/billing/credit-card.rst [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+You can add multiple credit cards per project. Only the credit card you have selected for the project will be charged.
```
Comment (Adrian-Turjak) [inline source/billing/credit-card.rst:63]: You can add multiple credit cards to your account. Only the credit card you have selected for the project will be charged, and different projects can be charged to different credit cards.

File: source/billing/credit-card.rst [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+You can also change back to Invoice as your payment method. The Catalyst Cloud will
+receive a notification request from your project.
+
+Before changing back to invoice method, we may reach out to you for a bit more of
+information on why you have decided to make that change.
+
+1) Login to the dashboard
+2) Go to Account Details, under Management
+3) Select the Project Billing Details panel
```
Comment (Adrian-Turjak) [inline source/billing/credit-card.rst:103]: tab not panel

**Comment by Adrian-Turjak** [comment]
Looks good to me, just also have @danielobyrne confirm and we can merge :)

File: source/billing.rst [hunk 6]
```diff
@@ -2,14 +2,22 @@
...
-We offer a billing API, that is easy to access and consume using a python
+There are many different ways to get your billing details using our system. The usage
+costs report shows you an estimate for your usage within the current region.
+
+It provides a breakdown per service, so you can quickly spot which services you are using
+the most and where your savings opportunities are. It also shows you the aggregate cost
+per month over the last twelve months, so you can identify usage patterns and project the
+expected usage for the current month.
+
+Our billing API provides an easy way to access and consume using a python
```
Comment (danielobyrne) [inline source/billing.rst:13]: This sentence doesn't sound right. (genuinely don't know but) does the API provide a way to access and consume the data or does the script access and consume data from the API?

File: source/billing/credit-card.rst [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+1) Go to Account Details, under Management
+2) Select the Project Billing Details tab
+
+.. image:: resources/billing-details.png
+   :align: center
+
+As you can see from the screenshot, your payment method should be as "Invoice".
+There are a few different ways to setup your credit card payment. You can also add a credit card without changing the payment method.
+
+To change to credit card and to set it up click on Change Payment method:
```
Comment (danielobyrne) [inline source/billing/credit-card.rst:27]: To change your active payment method to a credit card, you will need to click on  *Change payment method*:

File: source/billing/credit-card.rst [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+2) Select Add Credit Card
+3) It will prompt you do add a new credit card details
+4) You should also have a tick box allowing to assign this credit card to the project
+
+.. image:: resources/add-credit-card-updated.png
+   :align: center
+
+
+If while you were adding a new credit card you have not selected it as the primary
+option you still select it in the Project Billing Details tab:
```
Comment (danielobyrne) [inline source/billing/credit-card.rst:83]: you can*

**Comment by danielobyrne** [comment]
I've just left some comments on the changes, just some spelling or grammar changes since some of the sentences didn't seem to flow well. Other than that i trust that it all works well. I'll be happy to commit after those comments are looked at

**Comment by duduzis** [comment]
I have made the changes that Daniel suggested. I think it can be merged now. @chelios

**Pull request merged**